pkg/auth/api/api.go

```diff
@@ -390,3 +399,111 @@ func (s *authService) hasSystemAdminOrganization(orgs []*envproto.Organization)
+		s.logger.Info("Skip preparing demo user, password login config is not completed")
+	}
+	ctx := context.Background()
+	tx, err := s.mysqlClient.BeginTx(ctx)
+	if err != nil {
+		s.logger.Error("create mysql tx error", zap.Error(err))
+		return
+	}
+	now := time.Now()
+	err = s.mysqlClient.RunInTransaction(ctx, tx, func() error {
+		organizationStorage := envstotage.NewOrganizationStorage(tx)
+		_, err = organizationStorage.GetOrganization(ctx, s.config.PasswordConfig.Organization)
+		if err != nil && errors.Is(err, envstotage.ErrOrganizationNotFound) {
+			err = organizationStorage.CreateOrganization(ctx, &envdomain.Organization{
+				Organization: &envproto.Organization{
+					Id:          s.config.PasswordConfig.Organization,
+					Name:        "demo organization",
+					UrlCode:     "demo",
+					Description: "This organization is for demo user",
+					Disabled:    false,
+					Archived:    false,
+					Trial:       false,
+					CreatedAt:   now.Unix(),
+					UpdatedAt:   now.Unix(),
+					SystemAdmin: false,
+				}})
+			if err != nil {
+				return err
+			}
+		}
+		projectStorage := envstotage.NewProjectStorage(tx)
+		_, err = projectStorage.GetProject(ctx, s.config.PasswordConfig.Project)
+		if err != nil && errors.Is(err, envstotage.ErrProjectNotFound) {
+			err = projectStorage.CreateProject(ctx, &envdomain.Project{
+				Project: &envproto.Project{
+					Id:             s.config.PasswordConfig.Project,
+					Description:    "This project is for demo user",
+					Disabled:       false,
+					Trial:          false,
+					CreatorEmail:   demoUserEmail,
+					CreatedAt:      now.Unix(),
+					UpdatedAt:      now.Unix(),
+					Name:           "demo",
+					UrlCode:        "demo",
+					OrganizationId: s.config.PasswordConfig.Organization,
+				}})
+			if err != nil {
+				return err
+			}
+		}
+		environmentStorage := envstotage.NewEnvironmentStorage(tx)
+		_, err = environmentStorage.GetEnvironmentV2(ctx, s.config.PasswordConfig.Environment)
+		if err != nil && errors.Is(err, envstotage.ErrEnvironmentNotFound) {
+			err = environmentStorage.CreateEnvironmentV2(ctx, &envdomain.EnvironmentV2{
+				EnvironmentV2: &envproto.EnvironmentV2{
+					Id:             s.config.PasswordConfig.Environment,
+					Name:           "demo",
+					UrlCode:        "demo",
+					Description:    "This environment is for demo user",
+					ProjectId:      s.config.PasswordConfig.Project,
+					Archived:       false,
+					CreatedAt:      now.Unix(),
+					UpdatedAt:      now.Unix(),
+					OrganizationId: s.config.PasswordConfig.Organization,
+					RequireComment: false,
+				}})
+			if err != nil {
+				return err
+			}
+		}
+
+		return nil
+	})
+	if err != nil {
+		s.logger.Error("prepare demo user organization, project and environment error", zap.Error(err))
+		return
+	}
+	_, err = s.accountClient.GetAccountV2(ctx, &acproto.GetAccountV2Request{
+		Email:          demoUserEmail,
+		OrganizationId: s.config.PasswordConfig.Organization,
+	})
+	if err == nil {
```

Comment:
We should do so if the error is not found.
Like you did [here](https://clips.twitch.tv/DoubtfulAgreeableTroutBCouch-ODMobHQ8xGnLcWdj).